python/entities.py

```diff
@@ -42,16 +44,25 @@ def wrap_init(*args, **kwargs):
             object = args[0]
             for prop in object.json_properties:
                 setattr(object, prop, None)
-            for prop in object.recursive_properties:
+            for prop, remote_type in object.recursive_properties.items():
+                # dynamically import module if module specified as string
+                if isinstance(remote_type, str):
+                    remote_module_path = '.'.join(remote_type.split('.')[:-1])
+                    remote_cls_name = remote_type.split('.')[-1]
+                    remote_module = importlib.import_module(remote_module_path)
+                    object.recursive_properties[prop] = \
+                        getattr(remote_module, remote_cls_name)
                 setattr(object, prop, None)
 
         new_cls.__init__ = functools.update_wrapper(wrap_init, inner_init)  # type: ignore  # noqa
         for key, value in attrs.items():
-            if isinstance(value, Property) and getattr(value.remote_type,
-                                                       '_is_merchi_entity',
-                                                       False) \
-               and value.backref:
-
+            if isinstance(value, Property) and value.backref:
+                # TODO: 'Dynamic module load should also support backref'
+                if isinstance(value.remote_type, str):
+                    raise NotImplementedError('Backref has not been supported'
+                                              'for dynamic module reference')
```

Comment:
well, better fix this up

Reply:
yeah please let me do that in a separate branch, since I reckon it is "quite difficult to me"

Reply:
no TODO stuff in master please. if the branch isn't ready it isn't ready.